tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_sync_committee_updates.py

```diff
@@ -36,7 +32,39 @@ def test_sync_committees_progress(spec, state):
 
     # Can compute the third committee having computed final balances in the last epoch
     # of this `EPOCHS_PER_SYNC_COMMITTEE_PERIOD`
-    third_sync_committee = spec.get_sync_committee(state, 2 * spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD)
+    current_epoch = spec.get_current_epoch(state)
+    third_sync_committee = spec.get_sync_committee(state, current_epoch + 2 * spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD)
 
     assert state.current_sync_committee == second_sync_committee
     assert state.next_sync_committee == third_sync_committee
+
+
+@with_all_phases_except([PHASE0])
+@spec_state_test
+@with_configs([MINIMAL], reason="too slow")
+def test_sync_committees_progress_genesis(spec, state):
+    # Genesis epoch period has an exceptional case
+    spec.get_current_epoch(state)
+    assert spec.get_current_epoch(state) == spec.GENESIS_EPOCH
+
+    yield from run_sync_committees_progress_test(spec, state)
+
+
+@with_all_phases_except([PHASE0])
+@spec_state_test
+@with_configs([MINIMAL], reason="too slow")
+def test_sync_committees_progress_not_genesis(spec, state):
+    # Transition out of the genesis epoch period to test non-exceptional case
+    start_slot_of_next_period = state.slot + spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD * spec.SLOTS_PER_EPOCH
```

Comment:
we do happen to have the condition `state.slot == 0` at this point but i'm not sure if that's a solid invariant of the `spec_state_test` framework -- if not, could be worth explicitly checking: `assert spec.get_curent_epoch(state) == spec.GENESIS_EPOCH`